h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> If we've done it with vendored jsChannel, then it was a bad idea then too, in my opinion.

I have no objection against outlawing this practice in our projects.

What I don't like is talking about this current modification is if this was something unheard of, contrary to the fact that this is well withing our current practice. (Which might be misguided, as I am ready to admit.)

> The issue I have is not with modifying library code. The issue is the manner in which it's done. Any of the following would be better alternatives:
>
> - an explicit patch step when we build, with the differences maintained as a patch file
> - we monkeypatch the library at runtime
> - we move the library out of `vendor/` so that it's clear that we've forked it and are responsible for maintaining it
> - we fork the library ourselves and maintain our patches in the fork

Originally, I tried to do option 4.
Our fork is here:  https://github.com/csillag/jquery-scrollintoview/
What else would you like me to make this complete?

Or alternatively, we can do option 3.
Where would you like me to put it?